leader/leader.go

```diff
@@ -210,10 +198,7 @@ func isPodEvicted(pod corev1.Pod) bool {
 
 // getOperatorNamespace returns the namespace the operator should be running in.
 func getOperatorNamespace() (string, error) {
```

Comment:
Nit: have this take a path to make it testable (without resetting a global in tests)
```suggestion
func getOperatorNamespace(namespacePath string) (string, error) {
```